cache/http/http_test.go

```diff
@@ -2,229 +2,162 @@ package http
 
 import (
 	"bytes"
-	"crypto/sha256"
-	"encoding/hex"
-	"fmt"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
-	"net/url"
-	"os"
 	"strings"
 	"testing"
 
-	"github.com/buchgr/bazel-remote/cache"
-	"github.com/buchgr/bazel-remote/cache/disk"
-	"github.com/buchgr/bazel-remote/utils"
+	testutils "github.com/buchgr/bazel-remote/utils"
 )
 
-func TestProxyReadWorks(t *testing.T) {
-	// Test that reading a blob from a proxy works and also populates the local
```

Comment:
This functionality has moved to DiskCache, we should add a test there. And similarly for some of the other tests in this file.